tests/MyGame/MonsterExtraWithJvmStatic.kt

```diff
@@ -0,0 +1,167 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
All the files in this directory are automatically generated, so where does this file come from? If it was manually generated, let's delete it because it will not be updated.

Reply:
Additional tests put to code_generator.sh(and bat).
I choice monster_game.fbs
The result of them locate to tests/gen_jvmstatic and MonsterExtraWithJvmStatic.kt file not exist anymore.

Reply:
delete this file?

Reply:
Sorry, It's my mistake. I pushed commit again after remove this file.